proto/substrait/algebra.proto

```diff
@@ -891,6 +906,120 @@ message Expression {
     }
   }
 
+  // A window scalar function call.
+  message WindowScalarFunction {
```

Comment:
Instead of `WindowScalarFunction` maybe `WindowRelFunction`?  Also, can we put this message inside of `WindowRel`?

Reply:
Updated.